src/common/providers/tasks.ts

```diff
@@ -88,22 +88,72 @@ export interface TaskContext {
    * The result of decoding and verifying an ODIC token.
    */
   auth?: AuthData;
+
+  /**
+   * The name of the queue.
+   * Populated via the X-CloudTasks-QueueName header.
```

Comment:
Doc strings all look good!

Only question for me is whether "X-CloudTasks-QueueName" should be backticked as a literal.  Or is the idea here that we are suggesting variables, where "QueueName" will actually be a unique name, and X maybe a number?

Reply:
Yup, per offline, it sounds like we should backtick it for code font as in https://firebase.google.com/docs/functions/callable-reference#request_format_headers and elsewhere.